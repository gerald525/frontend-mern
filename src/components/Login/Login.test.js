
// import React from "react";
// import Enzyme, { shallow, render, mount } from "enzyme";
// import Adapter from "enzyme-adapter-react-16";
// import { createSerializer } from "enzyme-to-json";
// import sinon from "sinon";

// import Login from './Login';

// describe('<Login /> rendering', () => {
//     it("renders component ", () => {
//         const wrapper = render(
//             <BookForm 
//                 bookHandler={() => {}} 
//             />
//         );
//         expect(wrapper).toMatchSnapshot();
//     });