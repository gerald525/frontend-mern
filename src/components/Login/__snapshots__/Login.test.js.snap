// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> rendering renders Login component 1`] = `
initialize {
  "0": Object {
    "attribs": Object {},
    "children": Array [
      Object {
        "attribs": Object {},
        "children": Array [
          Object {
            "data": "Email",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "text",
          },
        ],
        "name": "label",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "attribs": Object {
            "id": "email",
            "name": "email",
            "type": "email",
          },
          "children": Array [],
          "name": "input",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": Object {
            "attribs": Object {},
            "children": Array [
              Object {
                "data": "Password",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "label",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": Object {
              "attribs": Object {
                "id": "password",
                "name": "password",
                "type": "password",
              },
              "children": Array [],
              "name": "input",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": Object {
                "attribs": Object {
                  "type": "submit",
                  "value": "Submit",
                },
                "children": Array [],
                "name": "input",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": [Circular],
                "type": "tag",
                "x-attribsNamespace": Object {
                  "type": undefined,
                  "value": undefined,
                },
                "x-attribsPrefix": Object {
                  "type": undefined,
                  "value": undefined,
                },
              },
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {
                "id": undefined,
                "name": undefined,
                "type": undefined,
              },
              "x-attribsPrefix": Object {
                "id": undefined,
                "name": undefined,
                "type": undefined,
              },
            },
            "parent": [Circular],
            "prev": [Circular],
            "type": "tag",
            "x-attribsNamespace": Object {},
            "x-attribsPrefix": Object {},
          },
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {
            "id": undefined,
            "name": undefined,
            "type": undefined,
          },
          "x-attribsPrefix": Object {
            "id": undefined,
            "name": undefined,
            "type": undefined,
          },
        },
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {},
        "x-attribsPrefix": Object {},
      },
      Object {
        "attribs": Object {
          "id": "email",
          "name": "email",
          "type": "email",
        },
        "children": Array [],
        "name": "input",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "attribs": Object {},
          "children": Array [
            Object {
              "data": "Password",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "label",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": Object {
            "attribs": Object {
              "id": "password",
              "name": "password",
              "type": "password",
            },
            "children": Array [],
            "name": "input",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": Object {
              "attribs": Object {
                "type": "submit",
                "value": "Submit",
              },
              "children": Array [],
              "name": "input",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {
                "type": undefined,
                "value": undefined,
              },
              "x-attribsPrefix": Object {
                "type": undefined,
                "value": undefined,
              },
            },
            "parent": [Circular],
            "prev": [Circular],
            "type": "tag",
            "x-attribsNamespace": Object {
              "id": undefined,
              "name": undefined,
              "type": undefined,
            },
            "x-attribsPrefix": Object {
              "id": undefined,
              "name": undefined,
              "type": undefined,
            },
          },
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {},
          "x-attribsPrefix": Object {},
        },
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {},
          "children": Array [
            Object {
              "data": "Email",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "label",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "tag",
          "x-attribsNamespace": Object {},
          "x-attribsPrefix": Object {},
        },
        "type": "tag",
        "x-attribsNamespace": Object {
          "id": undefined,
          "name": undefined,
          "type": undefined,
        },
        "x-attribsPrefix": Object {
          "id": undefined,
          "name": undefined,
          "type": undefined,
        },
      },
      Object {
        "attribs": Object {},
        "children": Array [
          Object {
            "data": "Password",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "text",
          },
        ],
        "name": "label",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "attribs": Object {
            "id": "password",
            "name": "password",
            "type": "password",
          },
          "children": Array [],
          "name": "input",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": Object {
            "attribs": Object {
              "type": "submit",
              "value": "Submit",
            },
            "children": Array [],
            "name": "input",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": [Circular],
            "type": "tag",
            "x-attribsNamespace": Object {
              "type": undefined,
              "value": undefined,
            },
            "x-attribsPrefix": Object {
              "type": undefined,
              "value": undefined,
            },
          },
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {
            "id": undefined,
            "name": undefined,
            "type": undefined,
          },
          "x-attribsPrefix": Object {
            "id": undefined,
            "name": undefined,
            "type": undefined,
          },
        },
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {
            "id": "email",
            "name": "email",
            "type": "email",
          },
          "children": Array [],
          "name": "input",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": Object {
            "attribs": Object {},
            "children": Array [
              Object {
                "data": "Email",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "label",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": [Circular],
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {},
            "x-attribsPrefix": Object {},
          },
          "type": "tag",
          "x-attribsNamespace": Object {
            "id": undefined,
            "name": undefined,
            "type": undefined,
          },
          "x-attribsPrefix": Object {
            "id": undefined,
            "name": undefined,
            "type": undefined,
          },
        },
        "type": "tag",
        "x-attribsNamespace": Object {},
        "x-attribsPrefix": Object {},
      },
      Object {
        "attribs": Object {
          "id": "password",
          "name": "password",
          "type": "password",
        },
        "children": Array [],
        "name": "input",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "attribs": Object {
            "type": "submit",
            "value": "Submit",
          },
          "children": Array [],
          "name": "input",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {
            "type": undefined,
            "value": undefined,
          },
          "x-attribsPrefix": Object {
            "type": undefined,
            "value": undefined,
          },
        },
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {},
          "children": Array [
            Object {
              "data": "Password",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "label",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": Object {
            "attribs": Object {
              "id": "email",
              "name": "email",
              "type": "email",
            },
            "children": Array [],
            "name": "input",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": [Circular],
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {},
              "children": Array [
                Object {
                  "data": "Email",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "label",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": [Circular],
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {},
              "x-attribsPrefix": Object {},
            },
            "type": "tag",
            "x-attribsNamespace": Object {
              "id": undefined,
              "name": undefined,
              "type": undefined,
            },
            "x-attribsPrefix": Object {
              "id": undefined,
              "name": undefined,
              "type": undefined,
            },
          },
          "type": "tag",
          "x-attribsNamespace": Object {},
          "x-attribsPrefix": Object {},
        },
        "type": "tag",
        "x-attribsNamespace": Object {
          "id": undefined,
          "name": undefined,
          "type": undefined,
        },
        "x-attribsPrefix": Object {
          "id": undefined,
          "name": undefined,
          "type": undefined,
        },
      },
      Object {
        "attribs": Object {
          "type": "submit",
          "value": "Submit",
        },
        "children": Array [],
        "name": "input",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {
            "id": "password",
            "name": "password",
            "type": "password",
          },
          "children": Array [],
          "name": "input",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": Object {
            "attribs": Object {},
            "children": Array [
              Object {
                "data": "Password",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "label",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": [Circular],
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "id": "email",
                "name": "email",
                "type": "email",
              },
              "children": Array [],
              "name": "input",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": [Circular],
              "parent": [Circular],
              "prev": Object {
                "attribs": Object {},
                "children": Array [
                  Object {
                    "data": "Email",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "label",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": [Circular],
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {},
                "x-attribsPrefix": Object {},
              },
              "type": "tag",
              "x-attribsNamespace": Object {
                "id": undefined,
                "name": undefined,
                "type": undefined,
              },
              "x-attribsPrefix": Object {
                "id": undefined,
                "name": undefined,
                "type": undefined,
              },
            },
            "type": "tag",
            "x-attribsNamespace": Object {},
            "x-attribsPrefix": Object {},
          },
          "type": "tag",
          "x-attribsNamespace": Object {
            "id": undefined,
            "name": undefined,
            "type": undefined,
          },
          "x-attribsPrefix": Object {
            "id": undefined,
            "name": undefined,
            "type": undefined,
          },
        },
        "type": "tag",
        "x-attribsNamespace": Object {
          "type": undefined,
          "value": undefined,
        },
        "x-attribsPrefix": Object {
          "type": undefined,
          "value": undefined,
        },
      },
    ],
    "name": "form",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {},
    "x-attribsPrefix": Object {},
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
